R/utilities.R

```diff
@@ -3015,6 +3020,116 @@ get_bams = function(sample,
   return(bam_details)
 }
 
+#' @title View a variant in IGV
+#'
+#' @description Load bam(s) and view the context around a mutation
+#'
+#' @details 
+#'
+#' @param this_mutation Specify the mutation of interest in MAF format.
+#' @param igv_port Specify the port IGV is listening on.
+#' @param socket 
+#' @param sort_by base, quality, sample or readGroup
+#'
+#' @return Path to file (.png).
+#'
+#' @import SRAdb
```

Comment:
What functions do we use from this package (SRAdb)? Do we want to load the package or just call the function from this package explicitly (SRAdb::function)?

Reply:
Everything IGV related needs this package. I'm not sure if we need to load it but feel free to test it if you want. testers of these new functions would be greatly appreciated. @vladimirsouza may also want to try this